myNotebook.py

```diff
@@ -77,105 +50,65 @@ class Label(tk.Label):
-            ttk.Checkbutton.__init__(self, master, **kw)
+    def __init__(self, master=None, **kw):
+        style = 'nb.TCheckbutton' if sys.platform == 'win32' else None
+        super().__init__(master, style=style, **kw)  # type: ignore
 
 
-class Radiobutton(sys.platform == 'darwin' and tk.Radiobutton or ttk.Radiobutton):  # type: ignore
+class Radiobutton(ttk.Radiobutton):
     """Custom t(t)k.Radiobutton class to fix some display issues."""
 
     def __init__(self, master: ttk.Frame | None = None, **kw):
-        if sys.platform == 'darwin':
-            kw['foreground'] = kw.pop('foreground', PAGEFG)
-            kw['background'] = kw.pop('background', PAGEBG)
-            tk.Radiobutton.__init__(self, master, **kw)
-        elif sys.platform == 'win32':
-            ttk.Radiobutton.__init__(self, master, style='nb.TRadiobutton', **kw)
-        else:
-            ttk.Radiobutton.__init__(self, master, **kw)
+        style = 'nb.TRadiobutton' if sys.platform == 'win32' else None
+        super().__init__(master, style=style, **kw)  # type: ignore
 
 
-class OptionMenu(sys.platform == 'darwin' and tk.OptionMenu or ttk.OptionMenu):  # type: ignore
-    """Custom t(t)k.OptionMenu class to fix some display issues."""
+class OptionMenu(ttk.OptionMenu):
+    """Custom ttk.OptionMenu class to fix some display issues."""
 
     def __init__(self, master, variable, default=None, *values, **kw):
-        if sys.platform == 'darwin':
-            variable.set(default)
-            bg = kw.pop('background', PAGEBG)
-            tk.OptionMenu.__init__(self, master, variable, *values, **kw)
-            self['background'] = bg
-        elif sys.platform == 'win32':
-            # OptionMenu derives from Menubutton at the Python level, so uses Menubutton's style
-            ttk.OptionMenu.__init__(self, master, variable, default, *values, style='nb.TMenubutton', **kw)
-            self['menu'].configure(background=PAGEBG)
-            # Workaround for https://bugs.python.org/issue25684
-            for i in range(0, self['menu'].index('end')+1):
-                self['menu'].entryconfig(i, variable=variable)
-        else:
-            ttk.OptionMenu.__init__(self, master, variable, default, *values, **kw)
-            self['menu'].configure(background=ttk.Style().lookup('TMenu', 'background'))
-            # Workaround for https://bugs.python.org/issue25684
-            for i in range(0, self['menu'].index('end')+1):
-                self['menu'].entryconfig(i, variable=variable)
+        style = 'nb.TMenubutton' if sys.platform == 'win32' else ttk.Style().lookup('TMenu', 'background')
+        menu_background = PAGEBG if sys.platform == 'win32' else ttk.Style().lookup('TMenu', 'background')
+
+        super().__init__(master, variable, default, *values, style=style, **kw)
+        self['menu'].configure(background=menu_background)
+
+        for i in range(self['menu'].index('end') + 1):
+            self['menu'].entryconfig(i, variable=variable)
```

Comment:
Warning, logic change: This code used to only run for ``linux``, now it runs for ``win32`` as well.

Reply:
Still runs for both ``linux`` and ``win32`` in the current code.